packages/jmix-react-antd/src/formatters/ant_to_jmixFront.ts

```diff
@@ -50,7 +50,7 @@ export function ant_to_jmixFront(
       return;
     }
 
-    if (propInfo && isToOneAssociation(propInfo) && typeof value === 'string') {
+    if (propInfo && isToOneAssociation(propInfo) && (typeof value === 'string' || typeof value === 'number')) {
```

Comment:
I've taken a quick look across the codebase and it seems that we expect `id` to be either `string` or `string | object` in other places.

I'd suggest to create an `EntityId` type and use it everywhere.

For example:

```
// Current code
export type HasId = {id: string | object};

// Will become
export type HasId = {id: EntityId};
```

Reply:
Also, it looks to me that we should check for `object` here as well:

```suggestion
    if (propInfo && isToOneAssociation(propInfo) && (typeof value === 'string' || typeof value === 'number' || typeof value === 'object')) {
```